### DISCUSSION TITLE
What's the different between pyside and pyqt builds?

### DISCUSSION THREAD

**Discussion by LuminarySage** [srevinsaju/guiscrcpy#350] [General]
When I go to the release page I see 2 different exe builds (ignoring the 32 and 64 bit versions) and I don't know which one to use

![image](https://user-images.githubusercontent.com/24306400/181935494-a4726b11-fe1f-473e-bd4d-9a2929e8beb2.png)

**Comment by srevinsaju** [comment]
Both of them uses different GUI APIs, See [this](https://www.pythonguis.com/faq/pyqt5-vs-pyside2/) link for more information. Technically, on the user side, it shouldn't matter, sometimes one of them do not work for certain OSes, while other might work.
